interp/notation.ml

```diff
@@ -2558,9 +2558,9 @@ let freeze ~marshallable =
   !delimiters_map, !scope_class_map,
   !prim_token_interp_infos, !prim_token_uninterp_infos,
   !entry_coercion_map, !entry_has_global_map,
-  !entry_has_ident_map)
+  !entry_has_ident_map, !notation_level_map)
```

Comment:
It shouldn't be defined as Summary.ref if it's included here

Reply:
also the stage of notation_level_map is synterp but this declare_summary is interp so it probably shouldn't be declared here (or one of the stages is incorrect)

Reply:
Right, I missed the redundancy, fixed. It is a priori ok that `notation_level_map` is synterp (used in `SYNTAX-EXTENSION`) and the other are interp (used in `NOTATION`).